test/notifications/display_test.dart

```diff
@@ -296,7 +375,31 @@ void main() {
         expectedTagComponent: 'dm:${message.allRecipientIds.join(",")}');
     }));
 
-    test('1:1 DM', () => awaitFakeAsync((async) async {
+    test('group DM: title updates with latest sender', runWithHttpClient((async) async {
+      await init();
+      final message1 = eg.dmMessage(from: eg.otherUser, to: [eg.selfUser, eg.thirdUser]);
+      final data1 = messageFcmMessage(message1);
+      final message2 = eg.dmMessage(from: eg.thirdUser, to: [eg.selfUser, eg.otherUser]);
+      final data2 = messageFcmMessage(message2);
+
+      final expectedTagComponent = 'dm:${message1.allRecipientIds.join(",")}';
+
+      await receiveFcmMessage(async, data1);
+      checkNotification(data1,
+        messageStyleMessages: [data1],
+        expectedIsGroupConversation: true,
+        expectedTitle: "${eg.otherUser.fullName} to you and 1 other",
+        expectedTagComponent: expectedTagComponent);
+
+     await receiveFcmMessage(async, data2);
+      checkNotification(data2,
```

Comment:
nit: Needs an extra space
```suggestion
      await receiveFcmMessage(async, data2);
      checkNotification(data2,
```